Packs/SentinelOne/Integrations/SentinelOne-V2/SentinelOne-V2.py

```diff
@@ -2624,21 +2624,18 @@ def connect_agent_to_network(client: Client, args: dict) -> Union[CommandResults
     agents_affected = raw_response.get('affected', 0)
 
     # Parse response into context & content entries
-    if agents_affected > 0:
```

Comment:
Why are we removing this check?

Reply:
@chloerongier, I believe I didn't explain myself clearly, my apologies.
In the original code the string `No agents were connected to the network.` is returned when no agents are effected. Is there a reason we should change this behavior?

Reply:
Indeed, in the original code this string is printed when no agent is affected. But it is only a print, no output **in the context** is returned.

If this command is used with **two** agents IDs as inputs, but only **one** is affected, the command output will still display both agent IDS in the context with their network status each, which will help determine which was indeed affected and which was not.

It makes sense to keep the same logic (context output with the IDs and network status) if no agent at all is affected.